COPY/usr/bin/manageiq-db-ready

```diff
@@ -23,10 +23,10 @@ def database_ready?(db_yaml, env = "production")
 end
 
 def manageiq_db_ready?(db_yaml, env = "production")
-  host, port, dbname, password = db_yaml[env].values_at("host", "port", "database", "password")
+  host, port, dbname, password, username = db_yaml[env].values_at("host", "port", "database", "password", "username")
   password = ManageIQ::Password.try_decrypt(password)
 
-  results = `PGPASSWORD=#{password} psql --host=#{host} --port=#{port || 5432} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
+  results = `PGPASSWORD=#{password} psql -U #{username} --host=#{host} --port=#{port || 5432} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
```

Comment:
Thanks for the PR!  

Perhaps this should be `-U #{username || 'root'}` in case the username isn't specified. 

What do you think @agrare?

Reply:
:+1: nice fix @johny5v and agreed @jrafanie we should default to `'root'` if nothing is present in the database.yml.

Reply:
good idea @jrafanie , I've updated the pull request code accordingly